### DISCUSSION TITLE
Operation on Master system 2

### DISCUSSION THREAD

**Discussion by leohenriquesilva** [darthcloud/BlueRetro#530] [General]
I have
8bitdo M30 - control
switch pro controller
Firmware v18.1 HW1 BlueRetro_parallel_2p or BlueRetro_genesis

Testing directly on the master system, regardless of the firmware, it synchronizes but does not work.
By mega drive in everdrive the controls work well when I open a master system game the control does not work.
Tried to configure control 3 buttons "Gamepad".
I selected the MD Genesis system.
Among other unsuccessful configurations, someone managed to play master system on the console or on the mega drive, could you please help me.

**Comment by Raton-Laveur** [comment]
Can confirm behaviour on BlueRetro firmware 1.83. 

Also stops working on Mega Drive with Master System Converter in GamePadAlt, when a real 6-button controller is OK in the same configuration.

**Comment by leohenriquesilva** [comment]
I tested now on the mega drive and the problem continues.
When selecting Master System games the control stops working.
firmware 1.83_hw1
Test other controls and it really won't.

**Comment by darthcloud** [comment]
I wont go into the technical detail but BlueRetro Genesis is only for genesis games.

For master systems support you need a different cable/adapter using the parallel 2p mode/firmware.

**Reply by leohenriquesilva** [reply]
Thank you.

**Comment by yuv422** [comment]
@darthcloud I've also got the same issue. I have a genesis cable on a Master System. I'm going to build a Parallel 2P cable. I noticed that you mention using the open drain PCB but the LS PCB is shown in the picture. Here https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-Cables-Build-Instructions#smd-cable-pcb-1

Should I use the Open Drain PCB OD v1.0? Or is the Level Shifter LS v1.2 board ok to use?

**Reply by darthcloud** [reply]
use open drain its the proper way, but both will work

**Reply by yuv422** [reply]
great. I'll use the OD one then. :)
